Adds an identifying mantle for Burzsiean IPC's

Karakamenk opened the pull request:
* Adds a new mantle to distinguish Burzsiean IPC's. It should be selectable in IPC Xenowear
![image](https://user-images.githubusercontent.com/65185737/165211698-db0bbd9b-cfb7-41a7-8ed3-e7dae45117bd.png)

Karakamenk commented:
!review

SleepyGemmy commented on code/modules/clothing/under/accessories/accessory.dm:
```diff
@@ -838,3 +838,12 @@
 	icon_state = "abnegtag"
+
+/obj/item/clothing/accessory/IPC_Mantle
+	name = "\improper Burzsiean Shoulder Mantle"
```
```suggestion
	name = "\improper Burzsiean shoulder mantle"
```

SleepyGemmy replied:
```suggestion
	name = "\improper Burzsiean shoulder mantle"
```

SleepyGemmy commented on code/modules/client/preference_setup/loadout/loadout_xeno/machine.dm:
```diff
@@ -166,4 +166,11 @@
+	whitelisted = list(SPECIES_IPC, SPECIES_IPC_G1, SPECIES_IPC_G2, SPECIES_IPC_XION, SPECIES_IPC_ZENGHU, SPECIES_IPC_BISHOP, SPECIES_IPC_SHELL)
+/datum/gear/accessory/ipcmantle
+	display_name = "Burzsiean shoulder mantle"
+	description = "A tarp identifying Hephaestus IPC's from the Burzsiean system. Operation history and specifications are printed underneath the tarp."
```
> Operation history and specifications are printed underneath the tarp.

maybe put this under the item description too?

Llywelwyn replied:
```suggestion
	description = "A tarp identifying Hephaestus IPC's from Burzsia. Operation history and specifications are printed underneath the tarp."
```

SleepyGemmy commented on code/modules/clothing/under/accessories/accessory.dm:
```diff
@@ -838,3 +838,12 @@
+
+/obj/item/clothing/accessory/IPC_Mantle
+	name = "\improper Burzsiean Shoulder Mantle"
+	desc = "A uniform mantle pulled over the right shoulder. The sigil of Burzsia is pressed into the front of it."
```
```suggestion
	desc = "An uniform mantle pulled over the right shoulder. The sigil of Burzsia is pressed into the front of it."
```

Llywelwyn commented on code/modules/client/preference_setup/loadout/loadout_xeno/machine.dm:
```diff
@@ -166,4 +166,11 @@
 	cost = 0
+	whitelisted = list(SPECIES_IPC, SPECIES_IPC_G1, SPECIES_IPC_G2, SPECIES_IPC_XION, SPECIES_IPC_ZENGHU, SPECIES_IPC_BISHOP, SPECIES_IPC_SHELL)
+/datum/gear/accessory/ipcmantle
+	display_name = "Burzsiean shoulder mantle"
```
```suggestion
	display_name = "Burzsian shoulder mantle"
```

its not actually written anywhere official, but Burzsian seems to be the agreed way of spelling this

Llywelwyn commented on code/modules/clothing/under/accessories/accessory.dm:
```diff
@@ -838,3 +838,13 @@
 	icon_state = "abnegtag"
+
+/obj/item/clothing/accessory/IPC_Mantle
+	name = "\improper Burzsiean shoulder mantle"
```
```suggestion
	name = "\improper Burzsian shoulder mantle"
```

Llywelwyn commented on code/modules/clothing/under/accessories/accessory.dm:
```diff
@@ -838,3 +838,13 @@
+/obj/item/clothing/accessory/IPC_Mantle
+	name = "\improper Burzsiean shoulder mantle"
+	desc = "An uniform mantle pulled over the right shoulder. The sigil of Burzsia is pressed into the front of it."
+	desc_fluff = "An identifying tarp issued to industrial IPC's that work without protective equipment on Burzsia. Tear resistant and chemically treated to withstand the planet of Burzsia I. More commonly seen on industrials or units that otherwise work inside of installations. Operation history and specifications are printed underneath the tarp, for field operators to quickly identify the unit in  ...
```
```suggestion
	desc_fluff = "An identifying tarp issued to industrial IPCs that work without protective equipment on Burzsia. Tear resistant and chemically treated to withstand the planet of Burzsia I. More commonly seen on industrials or units that otherwise work inside of installations. Operation history and specifications are printed underneath the tarp, for field operators to quickly identify the unit in the event it is damaged to the point where it cannot give diagnostics."
```

Llywelwyn reviewed (changes requested):
Just some spelling stuff in names/descs

Karakamenk commented:
Per the request of the IPC lore team, I've changed the tarp into metal plates, and refactored the accessory to behave like a poncho or cloak
![image](https://user-images.githubusercontent.com/65185737/166118014-fc68751e-df34-477e-a370-b11be8951ead.png)

JohnWildkins commented on code/modules/client/preference_setup/loadout/loadout_xeno/machine.dm:
```diff
@@ -166,4 +166,11 @@
+	whitelisted = list(SPECIES_IPC, SPECIES_IPC_G1, SPECIES_IPC_G2, SPECIES_IPC_XION, SPECIES_IPC_ZENGHU, SPECIES_IPC_BISHOP, SPECIES_IPC_SHELL)
+/datum/gear/accessory/ipcmantle
+	display_name = "Burzsian mantle"
+	description = "An uniform mantle identifying Hephaestus IPC's from Burzsia. Operation history and specifications are printed underneath the tarp."
```
```suggestion
	description = "A uniform mantle identifying Hephaestus IPC's from Burzsia. Operation history and specifications are printed underneath the tarp."
```
Dunno what dictionary Gemmy is using but `A uniform` is definitely correct here

JohnWildkins commented on code/modules/clothing/under/accessories/accessory.dm:
```diff
@@ -838,3 +838,14 @@
+
+/obj/item/clothing/accessory/poncho/IPC_Mantle
+	name = "\improper Burzsian shoulder mantle"
+	desc = "An uniform mantle made out of rudimentary metallic plates. The sigil of Burzsia is pressed into the front of it."
+	desc_fluff = "An uniform mantle of metallic plates that provide positronics in Burzsia cheap, rudimentary protection from industrial hazards and shrapnel; it's also been chemically treated to withstand the surface of Burzsia I. Operation history and specifications are printed underneath the back plate, as a failsafe for field operators to quickly identify the unit in the event it is damaged to t ...
```
```suggestion
	desc = "A uniform mantle made out of rudimentary metallic plates. The sigil of Burzsia is pressed into the front of it."
	desc_fluff = "A uniform mantle of metallic plates that provide positronics in Burzsia cheap, rudimentary protection from industrial hazards and shrapnel; it's also been chemically treated to withstand the surface of Burzsia I. Operation history and specifications are printed underneath the back plate, as a failsafe for field operators to quickly identify the unit in the event it is damaged to the point where sad information cannot be discerned through other means."
```

JohnWildkins commented on code/modules/client/preference_setup/loadout/loadout_xeno/machine.dm:
```diff
@@ -166,4 +166,11 @@
+/datum/gear/accessory/ipcmantle
+	display_name = "Burzsian mantle"
+	description = "An uniform mantle identifying Hephaestus IPC's from Burzsia. Operation history and specifications are printed underneath the tarp."
+	path = /obj/item/clothing/accessory/poncho/IPC_Mantle
```
```suggestion
	path = /obj/item/clothing/accessory/poncho/ipc_mantle
```
Technically not a hard and fast rule I don't think but the vast vast majority of items use all lowercase paths

JohnWildkins commented on code/modules/clothing/under/accessories/accessory.dm:
```diff
@@ -838,3 +838,14 @@
 	desc = "An embroidered patch indicating the wearer's blood type as AB NEGATIVE."
 	icon_state = "abnegtag"
+
+/obj/item/clothing/accessory/poncho/IPC_Mantle
```
```suggestion
/obj/item/clothing/accessory/poncho/ipc_mantle
```

NonQueueingMatt commented on code/modules/client/preference_setup/loadout/loadout_xeno/machine.dm:
```diff
@@ -166,4 +166,11 @@
 	description = "An identification card given to free IPC residents within the Republic of Biesel."
 	path = /obj/item/clothing/accessory/badge/passcard/synthetic
 	cost = 0
+	whitelisted = list(SPECIES_IPC, SPECIES_IPC_G1, SPECIES_IPC_G2, SPECIES_IPC_XION, SPECIES_IPC_ZENGHU, SPECIES_IPC_BISHOP, SPECIES_IPC_SHELL)
+/datum/gear/accessory/ipcmantle
```
```suggestion
	whitelisted = list(SPECIES_IPC, SPECIES_IPC_G1, SPECIES_IPC_G2, SPECIES_IPC_XION, SPECIES_IPC_ZENGHU, SPECIES_IPC_BISHOP, SPECIES_IPC_SHELL)

/datum/gear/accessory/ipcmantle
```

NonQueueingMatt commented on code/modules/clothing/under/accessories/accessory.dm:
```diff
@@ -838,3 +838,14 @@
+/obj/item/clothing/accessory/poncho/ipc_mantle
+	name = "\improper Burzsian shoulder mantle"
+	desc = "A uniform mantle made out of rudimentary metallic plates. The sigil of Burzsia is pressed into the front of it."
+	desc_fluff = "A uniform mantle of metallic plates that provide positronics in Burzsia cheap, rudimentary protection from industrial hazards and shrapnel; it's also been chemically treated to withstand the surface of Burzsia I. Operation history and specifications are printed underneath the back plate, as a failsafe for field operators to quickly identify the unit in the event it is damaged to th ...
```
```suggestion
	desc_fluff = "A uniform mantle of metallic plates that provide positronics in Burzsia cheap, rudimentary protection from industrial hazards and shrapnel; it's also been chemically treated to withstand the surface of Burzsia I. Operation history and specifications are printed underneath the back plate, as a failsafe for field operators to quickly identify the unit in the event it is damaged to the point where said information cannot be discerned through other means."
```
